Templates/GitlabCIPipeline-for-distributed-runner/.gitlab-ci.yml

```diff
@@ -2,21 +2,24 @@ variables:
     #Git strategy is set to "none" but overwrite later to "clone" in the jobs to create release candidate tag and release tag.
     GIT_STRATEGY: none
     GIT_CHECKOUT: "false"
+
     application: ${CI_PROJECT_NAME}
     wdEnvironmentFileIntegration: ${WAZI_DEPLOY_CONFIGDIR}/plum-samples/external-repos/environment-conf/python/Integration.yml
     wdEnvironmentFileAcceptance: ${WAZI_DEPLOY_CONFIGDIR}/plum-samples/external-repos/environment-conf/python/Acceptance.yml
     wdEnvironmentFileProduction: ${WAZI_DEPLOY_CONFIGDIR}/plum-samples/external-repos/environment-conf/python/Production.yml
-    packageName: ${application}.build-${CI_PIPELINE_ID}
+    
     # Directories on the GitLab runner environment to persist Wazi Deploy Evidence files and to create an Wazi Deploy Index
     wdEvidencesRoot: /var/work/wazi_deploy_evidences_gitlab/
     wdEvidencesIndex: /var/work/wazi_deploy_evidences_gitlab_index/
 
- # Working directory on USS 
+    # Working directory on USS 
     uniqueWorkspaceId: $PIPELINE_WORKSPACE/$CI_PROJECT_NAME/build-${CI_PIPELINE_ID}
-# Working directory on Linux machine
+    
+    # Working directory on Linux machine
     buildOutputDirectory: ${CI_PROJECT_DIR}/BUILD-OUTPUT
     deployOutputDirectory: ${CI_PROJECT_DIR}/DEPLOY-OUTPUT
-    baselineReferenceFile: $CI_PROJECT_DIR/$CI_PROJECT_NAME/application-conf/baselineReference.config
+    
+    # Variable options when running a pipeline
```

Comment:
On line 32: Let's set the default "releaseType":"minor" . Patch is reserved for release maintenance fixes.